package.json

```diff
@@ -54,6 +51,11 @@
                     ],
                     "default": false,
                     "description": "load all translation files configured in I18n.load_path"
+                },
+                "railsI18n.languageIdentifiers": {
+                    "type": "string",
```

Comment:
It may probably be nicer if the type would be array, then we could also add ```"minItems": 1``` and vscode could validate the setting. Although the downside to this is that apparently arrays can not yet be edited in the settings ui directly (I hope this feature will be added in the future), only via editing the json. What are your thoughts on this?

Reply:
I had it as an array originally, but looked through the source of a bunch of other extensions I have looking for configuration patterns looking for a way to add or remove a string value from the array. I certainly don't know a ton about the internals of VSCode extensions yet though. I learned just enough to make this change.

I personally feel this is more user-friendly than hand-editing the JSON for now, but would be glad to make the change if you think that would be better.

Reply:
I just looked through some of the extensions I've got installed, for example Typescript and SCSS seem both to have settings of type array too, but this may probably also be because these individual arrays may get quite extensive. 
Personally I think using an array may just be a bit cleaner, for now this may sadly be a bit less user-friendly, but I hope this is only until  the settings ui gets improved. At least as I added the setting in the json, vscode automatically populated the array with the default values, which is quite nice.
I'm also a bit concerned that if we used type string, we may be stuck with it, as I don't know if or how it could be migrated to another type in the future (if it can not be migrated, some users which don't use defaults may loose their settings then).

Reply:
I've updated this to use an `Array<string>` for configuration. It works great, and auto-populates the default values, however it isn't easily reviewable or editable from the settings screen. Maybe one day VSCode will add an interface for that.

Please let me know if there's anything else I can do to improve this.

Reply:
Nice, thanks! Yeah I hope they'll be adding it soon.